### ISSUE TITLE
Error: No such file or directory (os error 2)

### ISSUE THREAD

**Issue by StudioChew** [K4rakara/freshfetch#1]
I was trying to start it up for the first time, but I was met by the following error:
```
Error:
An I/O error occurred while trying to read from "/sys/devices/system/cpu/cpu0/cpufreq/bios_limit". Details:
No such file or directory (os error 2)
```

**Comment by K4rakara** [comment]
What operating system are you using (If Linux, which distribution)?

**Comment by K4rakara** [comment]
Should be fixed with v0.1.1!
